### DISCUSSION TITLE
How to locate the path of cc_binary when adding cc_binary as data of py_binary?

### DISCUSSION THREAD

**Discussion by FantasqueX** [bazelbuild/bazel#21783] [Q&A] [answered]
Referring to https://bazel.build/reference/be/python#py_binary_examples, I want to use cc_binary in py_binary, however, I don't know how to fill the path of `subprocess.run`. Simply `./cc_binary` doesn't work.

**Comment by meteorcloudy** [comment, marked as answer]
Please check https://github.com/bazelbuild/rules_python/tree/main/python/runfiles

**Reply by FantasqueX** [reply]
Got it! Thanks a lot!
